tests/test_filters.py

```diff
@@ -665,6 +686,123 @@ def test_glyph_cell_point_data(sphere):
         sphere.glyph(orient='vectors_points', scale='arr_cell', progress_bar=True)
 
 
+class InterrogateVtkGlyph3D:
+    def __init__(self, alg: pyvista._vtk.vtkGlyph3D):
+        self.alg = alg
+
+    @property
+    def input_data_object(self):
+        return pyvista.wrap(self.alg.GetInputDataObject(0, 0))
+
+    @property
+    def input_active_scalars_info(self):
+        return self.input_data_object.active_scalars_info
+
+    @property
+    def input_active_vectors_info(self):
+        return self.input_data_object.active_vectors_info
+
+    @property
+    def scaling(self):
+        return self.alg.GetScaling()
+
+    @property
+    def scale_mode(self):
+        return self.alg.GetScaleModeAsString()
+
+    @property
+    def scale_factor(self):
+        return self.alg.GetScaleFactor()
+
+    @property
+    def clamping(self):
+        return self.alg.GetClamping()
+
+    @property
+    def vector_mode(self):
+        return self.alg.GetVectorModeAsString()
+
+
+def test_glyph_settings(sphere):
+    sphere['vectors_cell'] = np.ones([sphere.n_cells, 3])
+    sphere['vectors_points'] = np.ones([sphere.n_points, 3])
+    sphere['arr_cell'] = np.ones(sphere.n_cells)
+    sphere['arr_points'] = np.ones(sphere.n_points)
+    sphere['active_arr_points'] = np.ones(sphere.n_points)
+    sphere['active_vectors_points'] = np.ones(sphere.n_points)
+
+    sphere.set_active_scalars('active_arr_points')
+    sphere.set_active_scalars('active_vectors_points')
```

Comment:
I assume this should be 
```suggestion
    sphere.set_active_vectors('active_vectors_points')
```

But given that this was unneeded for the existing tests, do we even need these?

Reply:
I've added a test which does actually require this. In words - "Test that when scale/orient are `True`, the already active scalars/vectors of the dataset are used for the glyphing."